### PR TITLE
[wasm]Fix the canvas measure size

### PR DESCRIPTION
**Pull request by Daoting**
GitHub Issue (If applicable): #5115 

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?

Canvas does not draw internal sub-canvas.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by jeromelaban** [comment]
@carldebilly

File: src/Uno.UI/UI/Xaml/Controls/Canvas/Canvas.Layout.cs [hunk 1]
```diff
@@ -23,16 +23,21 @@ public partial class Canvas : ICustomClippingElement
...
-					MeasureElement(child, new Size(double.PositiveInfinity, double.PositiveInfinity));
+					var measuredSize = MeasureElement(child, new Size(double.PositiveInfinity, double.PositiveInfinity));
+
+					maxHeight = Math.Max(maxHeight, measuredSize.Height + GetTop(child as DependencyObject));
+					maxWidth = Math.Max(maxWidth, measuredSize.Width + GetLeft(child as DependencyObject));
 				}
 			}
-			return new Size(0, 0);
+			// It will not be arranged if always return zero.
+			return new Size(maxWidth, maxHeight);
```
Comment (carldebilly) [inline src/Uno.UI/UI/Xaml/Controls/Canvas/Canvas.Layout.cs:40]: I just checked the source code of UWP/WinUI for the `<Canvas>` control (something I can't share here for obvious reasons) and the results on Windows is a constant `0, 0`.

So this fix is not appropriate and could break existing applications. Another fix should be found for #5115.

**Review by carldebilly — CHANGES_REQUESTED** [review]
Needs change

**Comment by MartinZikmund** [comment]
I think this PR is no longer relevant, as the underlying problem is now fixed.

**Pull request closed without merging**